<launch>
<!-- Arguments -->
  <arg name="scan_topic"     default="scan"/>
  <arg name="initial_pose_x" default="0.0"/>
  <arg name="initial_pose_y" default="0.0"/>
  <arg name="initial_pose_a" default="0.0"/>
<!-- Map Arguments -->
 <arg name="map_file" default="$(find imu_odom_combine)/maps/coverage_space2.map"/>
<!-- Map Server -->
 <node name="map_server" type="map_server" pkg="map_server" args="$(arg map_file)" />
<!-- fire up imu_combine-->
 <node pkg="imu_odom_combine" type="imu_odom_combine_node" name="imu_odom_combine" output="screen" >
  <!--inbound -->
  <param name="base_link_frame" value="/base_link" />
  <param name="imu_topic" value="/imu_data" />
  <param name="odom_topic" value="/wheel_diff_controller/odom" />
  <param name="publish_tf" value="true" />
  <!--outbound -->
  <param name="odom_comb_frame" value="/odom" />
  <param name="odom_combine_topic" value="odom_comb" />
</node>
 <!--fire up virtual amcl -->
<node pkg="fake_localization" type="fake_localization" name="fake_loc" output="screen" >
  <param name="global_frame_id" value="map" />
  <param name="odom_frame_id" value="odom" />
  <remap from="base_pose_ground_truth" to="odom_comb" />
</node>
<!--fireup ekf -->
<!--node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization_node" clear_params="true" output="screen">
  <rosparam command="load" file="$(find imu_odom_combine)/launch/ekf.yaml" />
</node-->


</launch>
